### PR TITLE
[45] Product Page using HTML

### PR DESCRIPTION
**Pull request by Tocyn**
html
- `Changes you made to given task`
Screen Shot 2020-09-26 at 10 55 53 PM

### CODE REVIEW

File: submissions/oluwatosinOke/html/tosin.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <div>
+        <ul>
+            <li> Samsung A20 </li>
+        </ul>
+    </div>
+</nav>
+
+<body>
+    <img src ="./images/ssa201.jpeg"> </img>
+</body>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:22]: You need to close `html` after the body tag @Tocyn

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/oluwatosinOke/html/tosin.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<p>
+    &#x20A6;75,000
+</p>
+<p> <strong> Product Description:</strong> </p>
+<p>
+<ol>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:29]: You did not close this tag @Tocyn

File: submissions/oluwatosinOke/html/tosin.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <li> RAM:4GB </li>
+    <li> Camera: Front and Rear </li>
+    </p>
+    <p> <strong> Similar Product from seller: </strong> </p>
+    <p>
+    <ol>
+        <li> Samsung A21 </li>
+        <li> Samsung A10s </li>
+        <li> Samsung A51 </li>
+    </Ol>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:41]: This tag should be in lowercase @Tocyn

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Hi,
Am I supposed to re submit

**Comment by ToluAkin** [comment]
@Tocyn you will make the necessary changes on the same branch. Commit the changes and push your changes.

**Review by adaugochi — COMMENTED** [review]
You are meant to use 2 images for this task as part of the success criteria

**Comment by Tocyn** [comment]
Done.

**Review by adaugochi — COMMENTED** [review]
Give a more descriptive and short name for these images

**Comment by adaugochi** [comment]
ssa201 doesn't mean anything to me

**Comment by Tocyn** [comment]
Do you mean I should rename the images??

**Comment by Tocyn** [comment]
I have renamed the images.
Thanks

**Comment by Tocyn** [comment]
Kindly review my PR

**Review by adaugochi — CHANGES_REQUESTED** [review]
You still have a comment on your PR

**Comment by Tocyn** [comment]
Please can you tell me which of the comment, because I have attended to all
comments to the best of my understanding.

Thanks

**Comment by adaugochi** [comment]
Can you check the `File changes` tab

**Comment by Tocyn** [comment]
Thanks Maryfaith.
I don't know the file changes you are referring to.
Do you want me to remove all spaces from the code?
Thanks again

File: submissions/oluwatosinOke/html/tosin.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+<nav>
+     <ul>
+          Samsung A20
+         <strong> &#x20A6;75,000</strong>
+    </ul>
+</nav>
+<body>
+    <p>
+        <img src ="./images/samsungA20frontview.jpeg"> </img>
+        <li>6.4" LCD display</li>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:19]: There is no parent `<ul>` for this `<li>` @Tocyn

File: submissions/oluwatosinOke/html/tosin.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+</header>
+<nav>
+     <ul>
+          Samsung A20
+         <strong> &#x20A6;75,000</strong>
+    </ul>
+</nav>
+<body>
+    <p>
+        <img src ="./images/samsungA20frontview.jpeg"> </img>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:18]: @Tocyn `img` tag is self-closing. There is no need for `</img>`

File: submissions/oluwatosinOke/html/tosin.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title> E COMMERCE </title>
+</head>
+<header>
+    <h1> MY FAVOURITE ONLINE STORE </h1>
+</header>
+<nav>
+     <ul>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:11]: You are not using this `<ul>` @Tocyn . You should remove it.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Thanks @ToluAkin.
I have effected the changes.

File: submissions/oluwatosinOke/html/tosin.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title> E COMMERCE </title>
+</head>
+<header>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:7]: @Tocyn header and nav tag should be inside the body tag

File: submissions/oluwatosinOke/html/tosin.html (hunk 2 above)
Comment (ToluAkin) [reply submissions/oluwatosinOke/html/tosin.html:27]: @Tocyn This has not been fixed

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Thanks ToluAkin, I don't understand what you mean by parent <ul>.

File: submissions/oluwatosinOke/html/tosin.html (hunk 4 above)
Comment (ToluAkin) [reply submissions/oluwatosinOke/html/tosin.html]: @Tocyn there should be a `ul` or `ol` tag as parent before using a `li`

**Comment by Tocyn** [comment]
Thank you ToluAkin, I have effected the said changes.

File: submissions/oluwatosinOke/html/tosin.html [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+<p>
+    <ol>
+        <li> Display:6.4-inch display </li>
+        <li> Storage:32GB </li>
+        <li> RAM:4GB </li>
+        <li> Camera: Front and Rear </li>
+    </ol>
+</p>
+    <p> <strong> Similar Product from seller: </strong> </p>
+<p>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:37]: @Tocyn Remove this paragraph tag it is not needed

File: submissions/oluwatosinOke/html/tosin.html [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+        </ul>
+    </p>
+   <p>
+    <img src ="./images/samsungA20backview.jpeg"> 
+        <ul>
+            <li>Backview with dual 13mp Camera and fingerprint scanner</li>
+        </ul>
+   </p> 
+<p> <strong> Product Description:</strong> </p>
+<p>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:28]: @Tocyn Remove this paragraph tag it is not needed

File: submissions/oluwatosinOke/html/tosin.html [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+          Samsung A20
+         <strong> &#x20A6;75,000</strong>
+</nav>
+    <p>
+        <img src ="./images/samsungA20frontview.jpeg">
+        <ul>
+            <li>6.4" LCD display</li>
+        </ul>
+    </p>
+   <p>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:21]: @Tocyn Remove this paragraph tag it is not needed

File: submissions/oluwatosinOke/html/tosin.html [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+<body>
+<header>
+    <h1> MY FAVOURITE ONLINE STORE </h1>
+</header>
+<nav>
+          Samsung A20
+         <strong> &#x20A6;75,000</strong>
+</nav>
+    <p>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/tosin.html:15]: @Tocyn Remove this paragraph tag it is not needed

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Thank you @ToluAkin. I have removed the said tags.

**Review by ToluAkin — APPROVED** [review]

**Comment by Tocyn** [comment]
Thank you for your tima and patience.

Warm Regards.

**Pull request merged**